api/src/load-data.ts

```diff
@@ -1,5 +1,5 @@
 import fs from "fs";
-export const dataLocation = "/data/data.json";
+export const dataLocation = "data/data.json";
```

Comment:
Always remember to change this back if not deployment will break!!!!

Reply:
SOZ I THOUGHT I CHANGED IT ARGH, will keep a fat reminder for it!

Reply:
Made sure this was done!